packages/codegen/src/EntityDbMetadata.ts

```diff
@@ -50,6 +58,7 @@ export type PrimitiveField = {
   notNull: boolean;
   derived: "orm" | "sync" | "async" | false;
   protected: boolean;
+  softDrop?: boolean;
```

Comment:
I'm pretty hard -1 on soft drop, I get your explanation of "it's a drop ... but soft" but still just throws me off, would prefer `ignored`. Or other suggestions.

Reply:
`ignored` makes sense.  It _could_ be used when adding a field also (for some reason?), in order to allow this logic to not be dependent on which direction we are going with it.

Reply:
I have updated this to be `ignore`. I believe it is ready to go now. 👍